### DISCUSSION TITLE
Is openssl+quic 3.0.9 released?

### DISCUSSION THREAD

**Discussion by xl32** [quictls/openssl#121] [Releases]
Is openssl+quic 3.0.9 released?
I can build it from openssl-3.0.9+quic branch, it passes my nginx tests. but there is no such release and tag.
Could you clarify if release is coming and what are the known problems?
Thanks!

**Comment by richsalz** [comment]
We distribute source.  Where are you finding a "release" ?  @wbl will add the missing tag.

**Reply by xl32** [reply]
@richsalz, that's easy, you previously had source github releases and I used them in my CD pipeline.

https://github.com/quictls/openssl/releases

**Reply by richsalz** [reply]
We've had some staffing changes, maybe we can restart doing this.  Someone want to volunteer?

**Comment by richsalz** [comment]
Please see #120 and continue the discussion there.
